custom-vu/src/main/kotlin/jces1209/vu/action/WorkAnIssue.kt

```diff
@@ -63,16 +63,20 @@ class WorkAnIssue(
 
     private fun linkIssue(issuePage: AbstractIssuePage, issuePrefixSearch: String) {
         val issueLinking = issuePage.linkIssue()
-        meter.measure(ISSUE_LINK) {
-            meter.measure(ISSUE_LINK_LOAD_FORM) {
-                issueLinking.openEditor()
-            }
-            meter.measure(ISSUE_LINK_SEARCH_CHOOSE) {
-                issueLinking.searchAndChooseIssue(issuePrefixSearch)
-            }
-            meter.measure(ISSUE_LINK_SUBMIT) {
-                issueLinking.submitIssue()
+        if (issueLinking.isLinkButtonPresent()) {
```

Comment:
Interesting. I guess this can be `false`, when the user doesn't have edit permissions.
This is how hardening looks like 👍